File: Datorklient/Aware2.0/src/model/Product.java
package model;

import java.time.LocalDate;

import javafx.beans.property.IntegerProperty;
import javafx.beans.property.ObjectProperty;
import javafx.beans.property. DoubleProperty;
import javafx.beans.property. IntegerProperty;
import javafx.beans.property. ObjectProperty;
import javafx.beans.property.SimpleDoubleProperty;
import javafx.beans.property.SimpleIntegerProperty;
import javafx.beans.property.SimpleStringProperty;
import javafx.beans.property. StringProperty;
import javafx.beans.property.StringProperty;

/**
 * Model class for a Person.
 *
 * @author Marco Jakob
 */
public  class Product {

    /**
     * Default constructor.
     */

    
    
    
    private int ProductId;
    private String Name;
    private String SKU;
    private int Quantity;
    private double Weight;
    private String StorageSpace;
    private String BarcodeNumber;
    private String ImageLocation;
    
    
    
    public void setProductId(int productId) {
        this.ProductId = productId;
    }
    public int getProductId() {
        return ProductId;
    }

    public void setName(String name) {
        this.Name = name;
    }


    public String getName() {
        return Name;
    }

    public void setSKU(String sku) {
        this.SKU = sku;
    }

    public String getSKU() {
        return SKU;
    }

    public void setQuantity(int quantity) {
        this.Quantity = quantity;
    }


    public int getQuantity() {
        return Quantity;
    }

    public double getWeight() {
        return Weight;
    }
     public void setWeight(double weight) {
         this.Weight = weight;
     }
    public String getStorageSpace() {
        return StorageSpace;
    }

    public void setStorageSpace(String storageSpace) {
        this.StorageSpace = storageSpace;
    }

    public String getBarcodeNumber() {
        return BarcodeNumber;
    }

    public void setImageLocation(String imageLocation) {
        this.ImageLocation = imageLocation;
    }
    public String getImageLocation() {
         return ImageLocation;
     }
    
    
    
    
    
    
    
    
    
    
    
}

File: Datorklient/AwareComputerClient/AwareComputerClient/Model/Product.cs

/**
 * Model class for a Product.
 *
 * @author Joakim Nilsson
 */
using System.ComponentModel;
namespace AwareComputerClient.Model
{
    public class Product
    {
        //Prevents generation of column
        [Browsable(false)]
        public int ProductId { get; set; }
        public string Name { get; set; }
        public string SKU { get; set; }
        public int Quantity { get; set; }
        public double Weight { get; set; }
        public string StorageSpace { get; set; }
        public string BarcodeNumber { get; set; }
        public string ImageLocation { get; set; }
        [Browsable(false)]
       public int LastInventory { get; set; }
    }
}

    